src/apps/intel_mp/test_10g_vmdq_pool_sel.sh

```diff
@@ -0,0 +1,69 @@
+#!../../snabb snsh
```

Comment:
The syntax highlighting is wrong. These files are Lua code, not shell scripts, so maybe they should have a `.lua` extension rather than `.sh`, even if they are executable.